sdk/Lusid.Sdk/Client/ApiClient.Dispose.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Lusid.Sdk.Client
+{
+    // DEV-7152: Bug in .NET Core 2.2 means any un-disposed HttpClients will leak sockets
+    // https://github.com/dotnet/corefx/pull/38499
+    // https://github.com/dotnet/runtime/issues/29327
+    // Use a finalizer to ensure RestClient (and thus HttpClient) instances are always disposed
+
+    public partial class ApiClient : IDisposable
```

Comment:
`ApiClient` is one of the auto-generated classes, and is regenerated each time. This implementation will need to be moved to the `utilities` folder which is excluded from being wiped on each run

Reply:
Thanks - move the file